demo/view-timeline-css/index.html

```diff
@@ -23,8 +23,23 @@
   width: 50px;
   height: 50px;
   animation-timeline: foo;
+  /* animation-time-range: enter 0% 100%; */
   animation: linear anim;
 }
+
+/*
```

Comment:
Add a comment that this is deliberately commented out.  Otherwise it could be deleted as part of a cleanup.  Here and above.